docker/build-website

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${COVID19_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+
+    echo -n \
+        "Usage: $(basename "$0")
+Build the website!
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        docker run -ti --rm \
+            -e COVID19_DEBUG=1 \
+            -e NODE_VERSION=10.19.0 \
+            -e RUBY_VERSION=2.6.2 \
+            -e PYTHON_VERSION=3.7 \
+            -v "$PWD/.:/opt/repo" \
+            -v "$PWD/.netlify-cache:/opt/buildhome/cache" \
+            -w /opt/build \
+            --entrypoint /bin/bash \
+            netlify/build:xenial \
+            -c "source /opt/buildhome/.rvm/scripts/rvm && cd /opt/repo && bundle install && /usr/local/bin/build ./scripts/cibuild && rsync -acvz --delete /opt/buildhome/repo/_site/ /opt/repo/_site/"
```

Comment:
@hectcastro as a note, changed this as I couldn't get the site to build correctly on a second run. The first run would work, but on a second run the `bundle install` step wouldn't happen and there would be missing ruby packages described in an error. If I removed `.netlify-cache`, this would solve the problem for the next run, but then subsequent runs would fail. Manually always installing gems with this addition seems to work.

Reply:
Interesting. I _think_ I may have addressed this in 0be1ded3c0d0e2acf323edf0b39d2d059e63270b. It seems like a setting Netlify was using to install the Ruby dependencies for the first time wasn't sticking across builds.